pyjion/jitinfo.h

```diff
@@ -1561,6 +1561,7 @@ class CorJitInfo : public ICorJitInfo, public JittedCode {
         memset(pEEInfoOut, 0, sizeof(CORINFO_EE_INFO));
         pEEInfoOut->inlinedCallFrameInfo.size = 4;
 #ifdef WINDOWS
+        pEEInfoOut->osPageSize = 1000; // Set to the windows default
```

Comment:
This should be 0x1000 (smaller value will work but you'll probe more often than necessary).